### PR TITLE
Add italian language for recommended.md

### PR DESCRIPTION
**Pull request by alessiobenvenuti**

### CODE REVIEW

File: recommended-it.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+- [PythonAnywhere.com](https://pythonanywhere.com)
+
+Hosting Python gratuito. Include un server Web con una facile configurazione e un database MySQL, se necessario. Esegui lavori cron, modifica i file e usa le console per Python, Bash e SQL.
+
+### Pagine Github 
+
+- [pages.github.com](https://pages.github.com/)
+
+hosting gratuito per siti statici creati con Jekyll o semplice HTML
```
Comment (MichaelCurrin) [inline recommended-it.md:46]:
```suggestion
Hosting gratuito per siti statici creati con Jekyll o semplice HTML.
```

File: recommended-it.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- Include un'anteprima accanto alla vista di modifica, a differenza dell'editor di Github che passa tra le modalità di modifica e anteprima.
+- Ha pulsanti per grassetto, corsivo, collegamento ipertestuale, ecc.
+- Si salva automaticamente ogni pochi minuti (eseguendo eventuali modifiche), il che elimina l'attrito dal salvataggio dei file.
+- Modifica e sincronizza i file sulle seguenti piattaforme
+- Dropbox
+- Github: quando li colleghi, puoi scegliere di includere o escludere l'accesso al repo privato.
+- Gitlab
+- Google Drive
+- Come modificare un repository:
+1. Vai a [stackedit.io] (https://stackedit.io/). Inizierai da una pagina di benvenuto.
```
Comment (MichaelCurrin) [inline recommended-it.md:66]:
```suggestion
1. Vai a [stackedit.io](https://stackedit.io/). Inizierai da una pagina di benvenuto.
```

File: recommended-it.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+Modifica i file markdown su Github usando il tuo browser.
+
+- Caratteristiche
+- Importa i tuoi repository Github
+- L'editor semplifica l'applicazione della formattazione utilizzando i pulsanti anziché digitarla.
+- Include un'anteprima accanto alla vista di modifica, a differenza dell'editor di Github che passa tra le modalità di modifica e anteprima.
+- Ha pulsanti per grassetto, corsivo, collegamento ipertestuale, ecc.
+- Si salva automaticamente ogni pochi minuti (eseguendo eventuali modifiche), il che elimina l'attrito dal salvataggio dei file.
+- Modifica e sincronizza i file sulle seguenti piattaforme
+- Dropbox
```
Comment (MichaelCurrin) [inline recommended-it.md:61]:
```suggestion
    - Dropbox
```

File: recommended-it.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+- Github: quando li colleghi, puoi scegliere di includere o escludere l'accesso al repo privato.
```
Comment (MichaelCurrin) [inline recommended-it.md:62]:
```suggestion
    - Github: quando li colleghi, puoi scegliere di includere o escludere l'accesso al repo privato.
```

File: recommended-it.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+- Gitlab
```
Comment (MichaelCurrin) [inline recommended-it.md:63]:
```suggestion
    - Gitlab
```

File: recommended-it.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+- Google Drive
```
Comment (MichaelCurrin) [inline recommended-it.md:64]:
```suggestion
    - Google Drive
```

File: recommended-it.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+- Come modificare un repository:
+1. Vai a [stackedit.io] (https://stackedit.io/). Inizierai da una pagina di benvenuto.
+2. Accedi con Github.
```
Comment (MichaelCurrin) [inline recommended-it.md:67]: 1 to 8 need to be indented 4 spaces or with a tab.

**Comment by MichaelCurrin** [comment]
@alessiobenvenuti Can I close this one? Or can you remove the extra file.

This has `resources-it-1.md` added now which doesn't look right.

But the newer PR has `resources-it.md` and that looks right #29 .

**Comment by alessiobenvenuti** [comment]
> @alessiobenvenuti Can I close this one? Or can you remove the extra file.
> 
> This has `resources-it-1.md` added now which doesn't look right.
> 
> But the newer PR has `resources-it.md` and that looks right #29 .

@MichaelCurrin , yes you can close, I've done a mess with PR☹️. I hope it's possible to close and merge the 2 file(resources-it.md and reccommended-it.md) to master.

**Pull request closed without merging**